common/include/pcl/Vertices.h

```diff
@@ -17,7 +18,7 @@ namespace pcl
     Vertices ()
     {}
 
-    std::vector<std::uint32_t> vertices;
+    std::vector<index_t> vertices;
```

Comment:
Note: ABI breakage here

Reply:
`Indices`

Reply:
I cant use an alias here right?

Reply:
Ambiguous question. `pcl::Indices` is an alias. Could you clarify the question?

Reply:
oh right, i was thinking in terms of `vector<index_t>`. Never mind.